### PR TITLE
Combined the ratings, credits, and links dataset.

### PR DESCRIPTION
**Pull request by andywang219**
Resulted in ~7570 data points. I'm not sure if I did the python modulo correctly. Please take a look.

### CODE REVIEW

File: main.ipynb [hunk 1]
```diff
@@ -18,6 +18,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "Couldn't Find dataset downloading now\n",
```
Comment (khelif96) [inline main.ipynb:21]: Clear the output before committing

File: Data_Clean_Up.ipynb [hunk 2]
```diff
@@ -0,0 +1,226 @@
+{
```
Comment (khelif96) [inline Data_Clean_Up.ipynb:1]: This shouldnt be a new jupyter notebook this should be a .py file it should be turned into a python module

File: movie_ratings.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    https://colab.research.google.com/drive/1yMfh75GfQG9prr565b-fkb19EQYoiQ70
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib as plt
+
+imUsingColab = False
+shouldDownload = True # Set this to False after the first run to avoid redownloading the dataset
+if shouldDownload:
```
Comment (khelif96) [inline movie_ratings.py:16]: This part isn't necessary

Comment (khelif96) [reply movie_ratings.py:16]: We aren't downloading any datasets in this file this should only handle cleaning.

File: movie_ratings.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    ratings_df = ratings_df.append(ratings_small_df, ignore_index=True)
+
+    # Making sure none of the movies was removed
+    print("number of unique moveies before groupby: ", ratings_df.movieId.unique().size)
+
+    # Calculate the average rating for each movie, drop userId and timestamp
+    ratings_df = ratings_df.drop(columns = ['userId', 'timestamp'])
+    ratings_df = ratings_df.groupby('movieId', as_index=False).mean()
+
+    print("number of unique moveies after groupby: ", ratings_df.movieId.unique().size)
```
Comment (khelif96) [inline movie_ratings.py:49]: Typos

File: movie_ratings.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```
Comment (khelif96) [inline movie_ratings.py:1]: Looks like your only linking the tables is there any cleaning happening or is the dataset already cleaned?

**Review by khelif96 — CHANGES_REQUESTED** [review]
Nice work so far but there are some changes that need to be done

File: main.ipynb (hunk 1 above)
Comment (andywang219) [reply main.ipynb:21]: Oh right, my bad, will fix

**Pull request closed without merging**